src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM.java

```diff
@@ -363,7 +363,7 @@ private void setRepositoryDetails(@CheckForNull String credentialsId, @Nullable
         @Inject
         private BitbucketPluginConfiguration bitbucketPluginConfiguration;
         @Inject
-        private JenkinsToBitbucketCredentials jenkinsToBitbucketCredentials;
+        private JenkinsToBitbucketCredentialsImpl jenkinsToBitbucketCredentials;
```

Comment:
Before we can continue with converting all interface to concrete classes, we should give a try in defining module. For example, 
``` 
public class JenkinsToBitbucketCredentialsModule extends com.google.inject.AbstractModule {
    @Override
    public void configure() {
        bind(JenkinsToBitbucketCredentials.class).to(JenkinsToBitbucketCredentialsImpl.class).in(com.google.inject.Singleton.class);
    }
}
```
as mentioned [here](https://wiki.jenkins.io/display/JENKINS/Dependency+Injection). If this does not work, then we should revert to this approach.